.github/workflows/release.yml

```
@@ -69,29 +72,31 @@ jobs:
       - name: Install cosign
         uses: sigstore/cosign-installer@v2
         with:
-          cosign-release: 'v1.12.1'
-      - name: Write cosign key to disk
-        id: write_key
-        run: echo "${{ secrets.COSIGN_KEY }}" > "/tmp/cosign.key"
+          cosign-release: 'v1.13.1'
       - name: Get Release Date
         id: release_date
         run: |
           RELEASE_DATE=$(date +"%y-%m-%d")
-          echo "::set-output name=RELEASE_DATE::${RELEASE_DATE}"
+          echo "RELEASE_DATE=${RELEASE_DATE}" >> $GITHUB_ENV
```

Let's use the brackets here too:

```suggestion
          echo "RELEASE_DATE=${RELEASE_DATE}" >> ${GITHUB_ENV}
```